demos/toolbar/custom-toolbar.html

```diff
@@ -36,7 +36,9 @@
         <section class="mdc-toolbar__section mdc-toolbar__section--align-end" role="toolbar">
           <a href="#" class="material-icons mdc-toolbar__icon demo-toolbar__icon--custom" aria-label="Download" alt="Download">file_download</a>
           <a href="#" class="material-icons mdc-toolbar__icon demo-toolbar__icon--custom" aria-label="Print this page" alt="Print this page">print</a>
-          <a href="#" class="material-icons mdc-toolbar__icon demo-toolbar__icon--custom" aria-label="Bookmark this page" alt="Bookmark this page">bookmark</a>
+          <span class="mdc-toolbar__icon demo-toolbar__icon--custom">
```

Comment:
We should probably keep this an `<a>` element:

```html
<a href="#" aria-label="Open in new window" alt="Open in new window">
```

Not only for consistency, but also because it makes the icon focusable and tabbable.